syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -57,6 +71,11 @@ class Syft internal constructor(
 
     private var requiresSpeedTest: Boolean = true
 
+    /**
+    * method to create a new Syft Worker Job.
+    * @throws IndexOutOfBoundsException if the there are too many jobs running.
```

Comment:
if the number of jobs exceed the [maxConcurrentJobs][StftConfiguration.maxConcurrentJobs]